packages/create-plugin/templates/github/ci/.github/workflows/ci.yml

```diff
@@ -60,7 +51,7 @@ jobs:
         if: steps.check-for-backend.outputs.has-backend == 'true'
         uses: actions/setup-go@v3
         with:
-          go-version: '1.19'
+          go-version: '1.20'
```

Comment:
should we use at least 1.20.4?

Reply:
from my testing this pulls the latest 1.20.x